docs/application/dotnet/guides/connectivity/internet-download.md

```diff
@@ -9,6 +9,10 @@ You can use the following Internet and content download features in your .NET ap
 
     You can get information about the connected network, and set up and configure network and socket connections. You can also track connection information, such as the size of the sent and received data.
 
+-   [WebView](webview.md)
```

Comment:
Is "WebView" written as a single word? 

Or Can write like this? 

# Web View

Reply:
In case of the module name, please follow the API reference.

In https://samsung.github.io/TizenFX/latest/api/Tizen.WebView.html, I can see it is written as WebView. So please use WebView.

Reply:
I agree with your suggestion, the reason I asked to change was only that I referenced the current live page. 
https://developer.tizen.org/development/guides/native-application/connectivity-and-wireless/internet-and-contents-downloads/web-view
Web View is written as two words.